universal-application-tool-0.0.1/app/services/question/QuestionDefinitionBuilder.java

```diff
@@ -36,6 +40,11 @@ public QuestionDefinitionBuilder(QuestionDefinition definition) {
     questionHelpText = definition.getQuestionHelpText();
     questionType = definition.getQuestionType();
     validationPredicatesString = definition.getValidationPredicatesAsString();
+
+    if (definition.getQuestionType().isMultiOptionType()) {
+      MultiOptionQuestionDefinition singleSelect = (MultiOptionQuestionDefinition) definition;
```

Comment:
Nit (blocking): the question may not always be single select even if it is now, and it doesn't seem to be necessary details here. Would you rename this variable to something like multiOption?

Reply:
Oops, great catch! I once called this a "single select" question and forgot to change this variable name